sdk/storage/storage-blob/src/AppendBlobClient.ts

```diff
@@ -8,10 +8,10 @@ import {
   isTokenCredential,
   isNode
 } from "@azure/core-http";
-
+import { CanonicalCode } from "@azure/core-tracing";
 import * as Models from "./generated/src/models";
 import { AbortSignalLike } from "@azure/abort-controller";
-import { BlobClient } from "./internal";
+import { BlobClient, CommonOptions } from "./internal";
```

Comment:
Did you forget to include changes in `internal.ts`?

Reply:
This got re-exported from `StorageClient` by this line:

```ts
export * from "./StorageClient";
```

Reply:
https://github.com/Azure/azure-sdk-for-js/pull/5357/files#diff-b9676d53b38575ed9e63e8c6b42b4c74

Reply:
> Oh, it's over here in StorageClient.ts:

Ah somehow I was looking for `internal.ts`.